### ISSUE TITLE
linter error when question mark at the end of a function name

### ISSUE THREAD

**Issue by lonnen** [SublimeLinter/SublimeLinter-jshint#23]
It's not valid JavaScript, but the following will cause an error in the SublimeLinter daemon instead of returning a bunch of warnings. If other lint errors were already in the file they will remain unchanged. They will not update further because of the error. Closing and loading the file results it no lint errors reported to the user. With debug mode on the linter reports to the console, and the error occurs before sublime can report it to the user.

```
function something?() {
    linter="should complain about this ->"
}
```

using OSX 10.8.5, Sublime 3 stable 3059, SublimeLint 3.2.11, SublimeLinter-jshint 1.1.4, jshint v2.4.3, and the lint on save setting.
